assets/sass/patterns/organisms/content-header.scss

```diff
@@ -25,6 +24,49 @@
 
 }
 
+// Content Header Grid Articles
+
+.content-header-grid {
+  padding: 0 24px;
+}
+
+.content-header-grid__main {
+  @include margin(24, "bottom");
+  @include padding(24, "bottom");
+  border-bottom: 1px solid $color-text-dividers;
+}
+
+.content-header-grid__side {
+  @include margin(24, "bottom");
+  @include padding(24, "bottom");
+  border-bottom: 1px solid $color-text-dividers;
+}
+
+@media only all and (min-width: 900px) {
+
+  .content-header-grid {
+    border-bottom: 1px solid $color-text-dividers;
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-gap: 24px;
+    max-width: 1114px;
```

Comment:
Do we not have a variable that can be used for max width? I'd imagine that it's the value needs to be the same for a large number of sections of the site.

Reply:
No we don't have this I think we can create a variable for it. There is two of these now I have implemented it into content header. The other is in info bars.

Reply:
There is max-site-width which is 1114px. There is one other place that uses 1114px which is in infobars so will change this as well.